gensrc/thrift/PlanNodes.thrift

```diff
@@ -448,6 +447,8 @@ struct TFileScanRangeParams {
     22: optional TTextSerdeType  text_serde_type 
     // used by flexible partial update
     23: optional string sequence_map_col
+    // if set true, be will be forced to use jni reader
+    24: bool force_jni_reader; 
```

Comment:
Should this behavior be specified in FE? Otherwise, when checking the plan, we won't know how many JNIs have been used.